### DISCUSSION TITLE
I need help to get Config Data of Board Name

### DISCUSSION THREAD

**Discussion by ngajar-feri** [khoih-prog/ESP_WiFiManager_Lite#15] [Q&A]
Hello... I need help.
How to get Config Data of Board Name, I Already input on portal. I Need to POST into My Database for unique each 8266.

In ESP_WiFiManager_Lite.h.
here the code.

 ![123](https://user-images.githubusercontent.com/68647766/188707619-6e979749-b402-441d-a16f-748d41354a5b.PNG)

**Comment by khoih-prog** [comment]
Use these functions

https://github.com/khoih-prog/ESP_WiFiManager_Lite/blob/bc8fb7ed5159e78f4cba35e07bf8cb18a7925320/src/ESP_WiFiManager_Lite.h#L973

https://github.com/khoih-prog/ESP_WiFiManager_Lite/blob/bc8fb7ed5159e78f4cba35e07bf8cb18a7925320/src/ESP_WiFiManager_Lite.h#L986

https://github.com/khoih-prog/ESP_WiFiManager_Lite/blob/bc8fb7ed5159e78f4cba35e07bf8cb18a7925320/src/ESP_WiFiManager_Lite.h#L999
